### DISCUSSION TITLE
abundance unit

### DISCUSSION THREAD

**Discussion by vrmarcelino** [micom-dev/micom#28] [Q&A] [answered]
Hi!

When modelling communities based on metagenome data, is it recommend using relative or absolute abundances? 

Thanks!!
Vanessa

**Comment by cdiener** [comment, marked as answer]
MICOM uses relative abundances internally, so if you give absolute ones they will just be converted to relative abundances in the construction step. So feel free to use what you'd like. Those are used as proxy for the relative amount of biomass (gDW) in lack of better data.

**Reply by vrmarcelino** [reply]
Okay, good to know.
Thanks for your reply!
